src/coins/coins.ts

```diff
@@ -1135,6 +1135,21 @@ export const basicCoins: BasicCoin[] = [
       },
     },
   },
+  // USDC.n
+  {
+    key: CoinKey.USDCn,
+    name: 'USDC via Noble',
```

Comment:
how about calling it `Noble USDC`, keep aligned with sei explorer
<img width="1446" height="689" alt="image" src="https://github.com/user-attachments/assets/f9fd19d3-b501-464b-9e85-c69d01b9786e" />

Reply:
<img width="460" height="88" alt="image" src="https://github.com/user-attachments/assets/246211e5-225a-47d1-82ef-d03398f680a6" />